src/index.html

```diff
@@ -9,6 +9,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="apple">Apple</a>
+          </li>
+          <li class="nav__item"><a href="samsung" class="nav__link">Samsung</a></li>
```

Comment:
каждый тег с новой строки
```suggestion
          <li class="nav__item">
            <a href="samsung" class="nav__link">Samsung</a>
          </li>
```